Docs 5537 add rtf to deployment strategies hn (#227)
mulesoft/docs-runtime-manager: modules/ROOT/pages/deploying-to-cloudhub.adoc

endif::[]
:keywords: cloudhub, cloud, deploy, manage, runtime manager, arm

image:logo-cloud-active.png[link="/runtime-manager/deployment-strategies#cloudhub", title="CloudHub"]
image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies#hybrid", title="Hybrid"]
image:logo-server-disabled.png[link="/runtime-manager/deployment-strategies#anypoint-platform-private-cloud-edition", title="Anypoint Platform PCE"]
image:logo-rtf-disabled.png[link="/runtime-manager/deployment-strategies#anypoint-runtime-fabric", title="Runtime Fabric"]

You can deploy Mule applications to CloudHub using:

* Runtime Manager
* Anypoint Studio
* CloudHub API
* CloudHub CLI
* Mule-Maven Plugin

== Naming an Application

> The application is always deployed with a public application name in the `cloudhub.io` domain. The application name must be globally unique across CloudHub to avoid domain conflicts. 

Reviewer: ```suggestion
Applications are always deployed with public application names in the `cloudhub.io` domain. Application names must be globally unique across CloudHub to avoid domain conflicts. 
```

> For this reason, create a naming convention for your applications. For example, you can always prefix your applications with `mycompany` and optionally a department name, for example `mycompany-mydept-myapplication`. You can then add your own DNS records to hide this complex application name, so for example you can route requests to `myapplication.mycompany.com` to `mycompany-mydept-myapplication.cloudhub.io`.

Reviewer: Note that the second sentence uses "for example" twice. Try to reword.

Also, thanks for your note in the style guide about Grammarly and the guidance to use ": for example,..." I am shocked--SHOCKED, I tell you!--and will look into this further!

Reviewer: Not too fond of the last sentence and "name, so for example you can..." Reword?
